cli/lib/site-utils.ts

```diff
@@ -83,20 +83,28 @@ export async function setupCustomDomain(
  * @param stoppedSiteId - The ID of the site that was just stopped (to exclude from the check)
```

Comment:
```suggestion
 * @param stoppedSiteIds - The ID of the site that was just stopped (to exclude from the check)
```

Looks like the param below is now in plural?

Reply:
Thanks Kat, sharp eyes, fixed